e2e-tests/portal/cypress/e2e/journeys/checker/cannot-submit-a-deal-with-newly-issued-facilities-that-have-cover-start-dates-before-min-submission-date/deal.js

```diff
@@ -17,7 +18,7 @@ const deal = {
   },
   maker: {
     _id: '5f3ab3f705e6630007dcfb25',
-    username: 'maker1@ukexportfinance.gov.uk',
+    username: BANK1_MAKER1.username,
```

Comment:
Again -- these can be rationalised more, however this is a large refactor so I've raised another ticket